external/source/HACKING.md

```diff
@@ -484,4 +484,4 @@ For running tests, the following dependencies are required:
 For compiling the C parts, you will need the package build dependencies:
 
     sudo dnf install dnf-utils python-srpm-macros
-    sudo dnf builddep --spec tools/cockpit.spec
\ No newline at end of file
+    sudo dnf builddep --spec tools/cockpit.spec
```

Comment:
Huh, what changed here? Apparently not trailing space..

Reply:
```diff
-    sudo dnf builddep --spec tools/cockpit.spec
\ No newline at end of file
+    sudo dnf builddep --spec tools/cockpit.spec
```

Reply:
Oh, ok -- thanks!